cmake/warnings.cmake

```diff
@@ -43,16 +43,26 @@ if (COMPILER_CLANG)
     add_warning(range-loop-analysis)
     add_warning(redundant-parens)
     add_warning(reserved-id-macro)
-    add_warning(shadow-field) # clang 8+
+    add_warning(shadow-field)
     add_warning(shadow-uncaptured-local)
     add_warning(shadow)
-    add_warning(string-plus-int) # clang 8+
+    add_warning(string-plus-int)
     add_warning(undef)
     add_warning(unreachable-code-return)
     add_warning(unreachable-code)
     add_warning(unused-exception-parameter)
     add_warning(unused-macros)
     add_warning(unused-member-function)
+    add_warning(unneeded-internal-declaration)
```

Comment:
We already have `-Weverything` by default. So, all this list has no effect.

Reply:
It still ok to put some warnings in this list, but keep in mind, that they were already enabled.